client/src/pages/dashboard/EventTypes.js

```diff
@@ -45,10 +60,14 @@ const EventTypes = () => {
               </Grid>
               <Grid container direction="column" item xs={11}>
                 <Grid item>
-                  <Typography>John Doe</Typography>
+                  {userData.user &&
```

Comment:
you might want to move this logic up and wrap it around the entire block for example

```
{userData.user && (
<Grid container direction="column" item xs={11}>
   <Grid item>
      <Typography>{userData.user.firstName} {userData.user.lastName}</Typography>
   </Grid>
   <Grid item>
       <Typography color="primary">calendly.com/{userData.user.firstName.toLowerCase()}-{userData.user.lastName.toLowerCase()}</Typography>
...
)}
```

Reply:
so that you don't have to repeat this logic check every time :)